### PR TITLE
20 user management UI

### PR DESCRIPTION
**Pull request by eddyhakimi**

### CODE REVIEW

File: src/deskstar-frontend/pages/userRequests.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import Head from "next/head";
```
Comment (n3rdc4ptn) [inline src/deskstar-frontend/pages/userRequests.tsx:1]: maybe we should put this page under /users/requests and the the overview under /users

**Review by n3rdc4ptn — CHANGES_REQUESTED** [review]
I would suggest changing the pages for a better route schema.
So that the overview page will be /users and the requests page will be /users/requests

File: src/deskstar-frontend/pages/userRequests.tsx (hunk 1 above)
Comment (eddyhakimi) [reply src/deskstar-frontend/pages/userRequests.tsx:1]: True, I will refactor this.

**Comment by eddyhakimi** [comment]
I adjusted the routing schema.

**Review by n3rdc4ptn — APPROVED** [review]

**Pull request merged**